### ISSUE TITLE
route53_zone: duplicate zones if value of zone does not end with a dot

### ISSUE THREAD

**Issue by sgzijl** [ansible/ansible-modules-extras#931]
##### Issue Type:

Bug report.
##### Ansible Version:

2.0 devel
##### Ansible Configuration:

```
inventory      = ec2.py
library        = ./ansible_modules/core:./ansible_modules/extras
roles_path    = roles

action_plugins     = ./ansible_plugins/action_plugins
callback_plugins   = ./ansible_plugins/callback_plugins
connection_plugins = ./ansible_plugins/connection_plugins
lookup_plugins     = ./ansible_plugins/lookup_plugins
vars_plugins       = ./ansible_plugins/vars_plugins
filter_plugins     = ./ansible_plugins/filter_plugins
```
##### Environment:

AWS & CentOS.
##### Summary:

The route53_zone module creates duplicate zones when the value of zone does not end with a dot. Route53 appends the dot automatically when not defined. 
##### Steps To Reproduce:

```
- name: Create example.tld
  route53_zone:
    state: present
    zone: example.tld
```
##### Expected Results:

Append a dot automatically when not defined (aws route53 api & route53.py behaviour) or validate input (zone) and fail.
##### Actual Results:

Duplicate zones with the same name.
##### Proposed fix:

To append a dot when not defined I changed line 80 to:

```
    # Ensure zone endswith a dot (#931)
    zone_in = module.params.get('zone').lower()
    if not zone_in.endswith('.'):
        zone_in += '.'
```

or (as is done in the route53.py module):

```
    zone_in = module.params.get('zone').lower()
    if zone_in[-1:] != '.':
        zone_in += "."
```

**Comment by sgzijl** [comment]
The fix could also be something like:
```
    zone_in = module.params.get('zone').lower()
    if zone_in[-1:] != '.':
        zone_in += "."
```
As is also done in the route53 module...

**Comment by sgzijl** [comment]
fixed in pull #963
